client/src/pages/ManageBookings/ManageBookings.tsx

```diff
@@ -0,0 +1,135 @@
+import { useState, useEffect } from 'react';
+import { useHistory } from 'react-router-dom';
+import PageContainer from '../../components/PageContainer/PageContainer';
+import { Box, Grid, Typography, Paper, CircularProgress } from '@mui/material';
+import useStyles from './useStyles';
+import RequestInfo from '../../components/RequestInfo/RequestInfo';
+import MyCalendar from '../../components/MyCalendar/MyCalendar';
+import { useAuth } from '../../context/useAuthContext';
+import getSitterRequests from '../../helpers/APICalls/sitterRequests';
+import { RequestApiData } from '../../interface/RequestApiData';
+import { format } from 'date-fns';
+import changeStatus from '../../helpers/APICalls/changeStatus';
+
+export default function ManageBookings(): JSX.Element {
+  const classes = useStyles();
+  const history = useHistory();
+  const { loggedInUser } = useAuth();
+  const { sitterRequests } = useAuth();
+  const { updateSitterRequestsContext } = useAuth();
+  const [dateValue, setDateValue] = useState<Date>(new Date(new Date().getFullYear(), new Date().getMonth(), 20));
+
+  const handleStatus = (requestId: string, newStatus: string): void => {
+    changeStatus(requestId, newStatus).then((res) => {
+      const index = sitterRequests.findIndex((request: RequestApiData) => {
+        return request._id === requestId;
+      });
+      if (index !== -1) {
+        const newSitterRequests = [...sitterRequests];
+        newSitterRequests[index].status = res.status;
+        updateSitterRequestsContext(newSitterRequests);
+      }
+    });
+  };
+
+  useEffect(() => {
+    if (loggedInUser) {
+      getSitterRequests(loggedInUser.id).then((res) => updateSitterRequestsContext(res));
+    }
+  }, [loggedInUser]); // eslint-disable-line react-hooks/exhaustive-deps
+
+  if (loggedInUser === undefined) return <CircularProgress />;
```

Comment:
This check and the one below it can be removed, as the protected route takes care of this functionality